dieses Build macht den Test kaputt:
```
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.714 sec <<< FAILURE! - in de.adorsys.oauth.sample.TestRevokeToken
testRevokeTokenNoCLientId(de.adorsys.oauth.sample.TestRevokeToken)  Time elapsed: 3.253 sec  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <302> doesn't match actual status code <500>.
	at de.adorsys.oauth.sample.TestRevokeToken.testRevokeTokenNoCLientId(TestRevokeToken.java:242)
```

--- a/oauth-client-jaas/src/main/java/de/adorsys/oauth/client/protocol/UserInfoResolver.java
+++ b/oauth-client-jaas/src/main/java/de/adorsys/oauth/client/protocol/UserInfoResolver.java
@@ -103,8 +103,8 @@ public class UserInfoResolver {
             httpGet.setHeader("Authorization", new BearerAccessToken(accessToken.getValue()).toAuthorizationHeader());
 
             HttpCacheContext context = HttpCacheContext.create();
-            try {
-                CloseableHttpResponse userInfoResponse = cachingHttpClient.execute(httpGet, context);
+            try (CloseableHttpResponse userInfoResponse = cachingHttpClient.execute(httpGet, context)){
+
 
                 //TODO mask accessToken
                 LOG.debug("read userinfo {} {}", OAuthCredentialHasher.hashCredential(accessToken.getValue()), context.getCacheResponseStatus());

--- a/oauth-client-jaas/src/main/java/de/adorsys/oauth/client/protocol/UserInfoResolver.java
+++ b/oauth-client-jaas/src/main/java/de/adorsys/oauth/client/protocol/UserInfoResolver.java
@@ -120,6 +120,7 @@ public class UserInfoResolver {
                 return UserInfo.parse(baos.toString());
             } catch (Exception e) {
                 // Auch bei einem Timeout die Exception weiterwerfen
+                LOG.warn("Exception beim accessing {}: ", userInfoEndpoint.toASCIIString(), e);
                 throw new IllegalStateException(e);
             }
         } catch (Exception e) {

